main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"net/http"
+)
+
+func main() {
+	/*
+			c := db.C("siafiles")
+			err = c.Insert(&Siafile{SiafilePath: "sia/1231231", Title: "First File"})
+
+			if err != nil {
+				log.Fatal(err)
+			}
+
+		result := new(Siafile)
+		err = c.Find(bson.M{"title": "First File"}).One(&result)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		fmt.Println("SiaFiles:", result.SiafilePath)
+	*/
+
+	sf := New()
```

Comment:
New what? `New` is a reasonable name when you're writing a library, because it prevents stuttering (e.g. `sunfish.NewSunfish` vs. `sunfish.New`). But if you're in `package main` it's just unclear.